rust/connlib/tunnel/src/device_channel/tun_windows.rs

```diff
@@ -134,8 +134,6 @@ impl Tun {
             self.remove_route(*old_route)?;
         }
 
-        // TODO: Might be calling this more often than it needs
-        flush_dns().expect("Should be able to flush Windows' DNS cache");
```

Comment:
Moved this to `on_update_resources`. I'm assuming we always get a resource update at startup, so this would be redundant anyway, and moving it brings Linux and Windows closer into alignment.